egs/fame/v1/run.sh

```diff
@@ -0,0 +1,300 @@
+          exp/ivectors_train exp/ivectors_fame_${task}_${subtask}_${sets}_enroll_male exp/ivectors_fame_${task}_${subtask}_${sets}_test_male $trials_male local/scores_gmm_2048_ind_male_${task}_${subtask}_${sets}
+              
+    done
+  done
+done
+
+for task in ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+      for year in _1t3 _4t10 _mt10; do
+
+        local/scoring_common.sh data/train data/fame_${task}_${subtask}_${sets}${year}_enroll data/fame_${task}_${subtask}_${sets}${year}_test \
+          exp/ivectors_train exp/ivectors_fame_${task}_${subtask}_${sets}${year}_enroll exp/ivectors_fame_${task}_${subtask}_${sets}${year}_test
+
+        trials_female=data/fame_${task}_${subtask}_${sets}${year}_test_female/trials
+        trials_male=data/fame_${task}_${subtask}_${sets}${year}_test_male/trials
+        trials=data/fame_${task}_${subtask}_${sets}${year}_test/trials
+
+        local/plda_scoring.sh data/train data/fame_${task}_${subtask}_${sets}${year}_enroll data/fame_${task}_${subtask}_${sets}${year}_test \
+          exp/ivectors_train exp/ivectors_fame_${task}_${subtask}_${sets}${year}_enroll exp/ivectors_fame_${task}_${subtask}_${sets}${year}_test $trials local/scores_gmm_2048_ind_pooled_${task}_${subtask}_${sets}${year}
+
+        local/plda_scoring.sh --use-existing-models true data/train data/fame_${task}_${subtask}_${sets}${year}_enroll_female data/fame_${task}_${subtask}_${sets}${year}_test_female \
+          exp/ivectors_train exp/ivectors_fame_${task}_${subtask}_${sets}${year}_enroll_female exp/ivectors_fame_${task}_${subtask}_${sets}${year}_test_female $trials_female local/scores_gmm_2048_ind_female_${task}_${subtask}_${sets}${year}
+
+        local/plda_scoring.sh --use-existing-models true data/train data/fame_${task}_${subtask}_${sets}${year}_enroll_male data/fame_${task}_${subtask}_${sets}${year}_test_male \
+          exp/ivectors_train exp/ivectors_fame_${task}_${subtask}_${sets}${year}_enroll_male exp/ivectors_fame_${task}_${subtask}_${sets}${year}_test_male $trials_male local/scores_gmm_2048_ind_male_${task}_${subtask}_${sets}${year}
+
+      done
+    done
+  done
+done
+
+# Calculating EER 
+
+echo "Calculating EER.."
+
+for task in complete ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+
+        trials=data/fame_${task}_${subtask}_${sets}_test/trials
+        echo "GMM-$num_components EER for fame_${task}_${subtask}_${sets}"
+        for x in ind; do
+          for y in female male pooled; do
+            echo "python local/prepare_for_eer.py $trials local/scores_gmm_${num_components}_${x}_${y}_${task}_${subtask}_${sets}/plda_scores"
+            eer=`compute-eer <(python local/prepare_for_eer.py $trials local/scores_gmm_${num_components}_${x}_${y}_${task}_${subtask}_${sets}/plda_scores) 2> /dev/null`
```

Comment:
I think the prepare_for_eer.py script can be replaced with the awk/paste command I listed earlier... If it isn't possible then don't worry about it, though.

Reply:
Well, I would prefer keeping it as it is then as now it is consistent with sre examples.